Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/omf/OmfSymbol.java

```diff
@@ -26,6 +28,15 @@ public class OmfSymbol {
 	private long offset;
 	private Address address;
 
+	/**
+	 * These names were taken from the OpenWatcom source code
+	 * https://github.com/open-watcom/open-watcom-v2/blob/master/bld/watcom/h/fppatche.h
+	 */
+	private final Set<String> FLOATINGPOINT_SPECIALNAMES = Set.of(
+		"FIWRQQ", "FIDRQQ", "FIERQQ", "FICRQQ", "FJCRQQ", "FISRQQ",
```

Comment:
still need "static".

Reply:
and there are some places that could use reformatting use the eclipse code formatter settings mentioned above.